pkg/files.go

```diff
@@ -383,3 +383,10 @@ func UniqueFileInfos(fileInfos []FileInfo) []FileInfo {
 	}
 	return list
 }
+
+// func UniqueFileInfos(fileInfos []FileInfo) []FileInfo {
```

Comment:
You kept them here for a later use ?

Reply:
Removed them. I tested the asis and Tobe first, as there were no tests for that.